src/binary-encoding.adoc

```diff
@@ -841,6 +841,10 @@ though it is less compact than opcodes `0x73`-`0x77` above.
 |===
 _{asterisk} Serialized size in bytes does not include the opcode._
 
+The body of short-form timestamps are encoded as a `FixedUInt` of the specified size of the op-code.  This integer is
+then partitioned in bit-fields for the timestamp's subfields.  Note that endian does not apply here because the
+bit-fields are defined over the body interpreted as an _integer_.
+
```

Comment:
```suggestion
The body of short-form timestamps are encoded as a `FixedUInt` of size specified by the opcode.  This integer is
then partitioned into bit-fields representing the timestamp's subfields.  Note that endianness does not apply here because the
bit-fields are defined over the body interpreted as an _integer_.

```

I think the "here" in "does not apply here" could be spelled out a bit more.